content/news/XXXX-XX-XX-dvs-internals-pt1.md

```diff
@@ -0,0 +1,115 @@
+title: "How Does Timecode Vinyl Actually Work? (Pt. 1)"
+authors: Jan Holthuis
+tags: 2.3, timecode, dvs, vinyl control, serato, cd
+status: draft
+
+
+Since the dawn of DJing, spinning vinyl records never really went out of fashion in the DJ community.
+Even when CDs became popular in 90s and most music listeners happily phased out their record players, the classical "two turntables and a mixer" setup stayed the epitome of DJing.
+In recent years, digital DJ controllers became more popular, but there are still many DJs that stick with their Technics SL-1210s turntables.
+
+There's a reason for that: the tactile feeling of manipulating music with your hands is still very different from what most controllers can do.
+Motorized controllers like the Rane One try to provide a similar workflow, but it's still to be seen if they will have a notable impact on the prevalence of vinyl.
+
+On the other hand, going digital has its benefits:
+Jumping to different positions inside a track using hotcues just isn't possible with vinyl records, and carrying around heavy crates of records isn't the best you can do for your back.
```

Comment:
not quite true!  DJs can use stickers on vinyl to guide the needle to very precise positions on the record.  So it's possilble, but it's not practical.